README.md

```diff
@@ -135,33 +136,29 @@ xmrigDaemon -o pool.minemonero.pro:5555 -u YOUR_WALLET -p x -k --cc-url=IP_OF_CC
   -l, --log-file=FILE                   log all output to a file
   -h, --help                            display this help and exit
   -V, --version                         output version information and exit
+  -v, --av=N                            DEPRECATED - algorithm variation, 0 auto select
+      --doublehash-thread-mask          DEPRECATED - same as multihash-thread-mask
 
 ```
 
 Also you can use configuration via config file, default **[config.json](https://github.com/Bendr0id/xmrigCC/wiki/Config-XMRigDaemon)**. You can load multiple config files and combine it with command line options.
 
-## Algorithm variations
-* `--av=1` For CPUs with hardware AES.
-* `--av=2` Lower power mode (double hash) of `1`.
-* `--av=3` Software AES implementation.
-* `--av=4` Lower power mode (double hash) of `3`.
-
-## Doublehash thread Mask (only for low power mode (av=2 and av=4))
-With this option you can limit doublehash to the given threads (mask). This can significantly improve your hashrate by using unused l3 cache. The default is to run av2/av4 mode on all threads.
+## Multihash thread Mask (only for multihash-factor > 1)
+With this option you can limit multihash to the given threads (mask). This can significantly improve your hashrate by using unused l3 cache. The default is to run av2/av4 mode on all threads.
```

Comment:
Add: Short description of --multihash-factor

Outdated: "The default is to run av2/av4 mode on all threads."